src/test/resources/org/opensearch/agent/tools/register_flow_agent_of_ragtool_with_neural_query_type_request_body.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Test_Agent_For_RagTool",
+  "type": "flow",
+  "description": "this is a test flow agent in flow",
+  "tools": [
+    {
+      "type": "RAGTool",
+      "description": "A description of the tool",
+      "parameters": {
+        "embedding_model_id": "<MODEL_ID>",
+        "inference_model_id": "c8NwFo0BY4jgIz2mgLNH",
```

Comment:
why hard code inference model id here

Reply:
fair point, for this agent which didn't enable_content_generation, we actually doesn't need inference_model_id, I will remove this line ` "inference_model_id": "c8NwFo0BY4jgIz2mgLNH"`.